docs/config/README.md

```diff
@@ -637,11 +637,12 @@ it would have been `nixpkgs/pkgs`.
 | ------------------------ | ----------------------------------------------- | -------------------------------------------------------------------------------- |
 | `truncation_length`      | `3`                                             | The number of parent folders that the current directory should be truncated to.  |
 | `truncate_to_repo`       | `true`                                          | Whether or not to truncate to the root of the git repo that you're currently in. |
-| `format`                 | `"[$path]($style)[$lock_symbol]($lock_style) "` | The format for the module.                                                       |
+| `format`                 | `"[$truncated]($style)[$path]($style)[$read_only]($read_only_style) "` | The format for the module.                                |
 | `style`                  | `"bold cyan"`                                   | The style for the module.                                                        |
 | `disabled`               | `false`                                         | Disables the `directory` module.                                                 |
 | `read_only_symbol`       | `"🔒"`                                          | The symbol indicating current directory is read only.                            |
 | `read_only_symbol_style` | `"red"`                                         | The style for the read only symbol.                                              |
+| `truncation_symbol`      | `"…/"`                                            | The symbol indicating path truncation.                                         |
```

Comment:
Not sure whether this should be shown by default. When the default path separator changes to `\` on windows this will need to become platform dependent too and I don't think this is the default behavior for most shells, at least it's not in spaceship.

Reply:
Will change to just "" and expand the docs.

Reply:
FWIW, the path's get converted to `/` anyways on windows. See the first few windows tests.